docs/resources/task_grant.md

```diff
@@ -53,6 +53,6 @@ resource "snowflake_task_grant" "grant" {
 Import is supported using the following syntax:
 
 ```shell
-# format is database_name|schema_name|task_name|privilege|with_grant_option|on_future|on_all|roles"
-terraform import snowflake_task_grant.example "MY_DATABASE|MY_SCHEMA|MY_TASK|OPERATE|false|false|false|role1,role2"
+# format is database_name|schema_name|task_name|privilege|with_grant_option|on_future|roles"
```

Comment:
are you removing the changes from #2592 ?

Reply:
Didn't notice it was previously added only to .md file so It was re-generated to the wrong format. Fixed import.sh file, so it generated as it should be.